### PR TITLE
BDLaethf35

### PR DESCRIPTION
**Pull request by thea-m**
Hi, 
here is one of the new additions. Not all leaves could be photographed, so the textual description might be completed at a later point.
Have fun!

### CODE REVIEW

**Review by gnizla — APPROVED** [review]

File: OxfordBodleian/BodNewMSS/BDLaethf35.xml [hunk 1]
```diff
@@ -34,9 +37,97 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                    <persName ref="PRS14002WaldaS">ወልደ፡ ሥላሴ፡</persName> ለዓለመ<supplied reason="omitted">፡</supplied></hi> 
+                                ዓለም፡ አሜን፡ 
+                                ጐሥዓ፡ ልብ<cb n="b"/><hi rend="rubric">የ፡ ቃለ፡ ሠናየ፡ ጐሥዓ፡ ልብየ፡</hi> ቃለ፡ ሠናየ፡ ወአነ፡ አየድዕ፡
+                                ቅዳ<hi rend="rubric">ሴሃ፡ ለማርያም<supplied reason="omitted">፡</supplied> አኮ፡
+                                    በአብዝኆ፡</hi> አላ፡ በአውህዶ<supplied reason="omitted">፡</supplied> ወአነ፡ አየድዕ<supplied reason="omitted">፡</supplied> ውዳሴሃ፡ ለድንግል።
+                            </incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i3">
+                            <locus to="107ra"/>
```
Comment (MassimoVilla) [inline OxfordBodleian/BodNewMSS/BDLaethf35.xml:95]: `<locus to="107ra"/>`? Text 3.1 is said to begin on fol. 105rb.

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Thea. Two very minimal comments

File: OxfordBodleian/BodNewMSS/BDLaethf35.xml (hunk 1 above)
Comment (thea-m) [reply OxfordBodleian/BodNewMSS/BDLaethf35.xml:95]: The problem is that the beginning of the text was not photographed. 3.1. ends on 105rb, but we don't have its beginning, so I added only the value for the text's ending. I added a comment in the record to make it clearer.

**Comment by thea-m** [comment]
Thank you, Massimo!

**Review by ElianaDalSasso — APPROVED** [review]
Thank you Thea! All good!

**Pull request merged**